source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -0,0 +1,189 @@
+array elements to update. 
+
+To specify elements in an array with positional operators, use the **dot
+notation**. Dot notation is a property access syntax for navigating JSON
+objects. 
+
+For additional information, see the server manual entry on
+:manual:`dot notation </core/document/#std-label-document-dot-notation>`.
+
+The First Array Element
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To update the first element that matches your query filter, use the
+positional ``$`` operator. The array field must appear as part of your
+query document to use the positional ``$`` operator. 
+
+Example
+```````
+
+The following example performs these actions:
+
+- Query for a document with a ``qty`` field containing the value "18"
+- Decrement the first array value that matches the query document by "3"  using the ``inc()`` method of the ``Updates`` builder and a positional ``$`` operator 
+- Update and retrieve the document
+
+.. literalinclude:: /includes/fundamentals/code-snippets/UpdateArray.java
+   :language: java
+   :dedent:
+   :start-after: begin updateValueExample
+   :end-before: end updateValueExample
+
+The example above updates the original document to the following state:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "color": "green", "qty": [8, 12, 15] }
+
+For additional information on the operator and method mentioned in this
+section, see the following resources: 
+
+- :manual:`Positional $ Operator </reference/operator/update/positional>` Server Manual Entry
+- :java-core-api:`inc() <com/mongodb/client/model/Updates.html#inc(java.lang.String,java.lang.Number)>` API Documentation
+
+All Array Elements
+~~~~~~~~~~~~~~~~~~
+
+To update all elements in an array that match your query filter, use the
+all positional ``$[]`` operator. 
+
+Example
+```````
+
+The following example performs these actions: 
+
+- Query for the sample document
+- Multiplies each value by "2"  using the ``mul()`` method of the ``Updates`` builder 
+- Updates all elements in the ``qty`` array using an all ``$[]`` positional operator
+
+.. literalinclude:: /includes/fundamentals/code-snippets/UpdateArray.java
+   :language: java
+   :dedent:
+   :start-after: begin updateAllElementsExample
+   :end-before: end updateAllElementsExample
+
+The example above updates the original document to the following state:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "color": "green", "qty": [16, 24, 36] }
+
+For additional information on the operator and method mentioned in this
+section, see the following resources: 
+
+- :manual:`All Positional $[] Operator </reference/operator/update/positional-all/>` Server Manual Entry
+- :java-core-api:`mul() <com/mongodb/client/model/Updates.html#mul(java.lang.String,java.lang.Number)>` API Documentation
+
+Multiple Array Elements
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To update multiple array elements that match your query filter, use the
+filtered positional ``$[<identifier>]`` operator. You must include an
```

Comment:
```suggestion
To update array elements that match a filter, use the
filtered positional ``$[<identifier>]`` operator. You must include an
```
Issue: This is similar to a comment from All Array Elements from earlier in this round of reviews. I do not believe the filtered query operator relates to the query filter from the find operation (in the example below the query document in the find operation does not specify an array).